addons/website_crm_partner_assign/views/crm_lead_views.xml

```diff
@@ -101,6 +101,30 @@
             </field>
         </record>
 
+        <record id="crm_opportunity_report_view_pivot_lead" model="ir.ui.view">
+            <field name="name">crm.opportunity.report.view.pivot.lead.inherit.partner_assign</field>
+            <field name="model">crm.lead</field>
+            <field name="inherit_id" ref="crm.crm_opportunity_report_view_pivot_lead"/>
+            <field name="arch" type="xml">
+                <xpath expr="//pivot" position="inside">
+                    <field name="partner_latitude" invisible="1"/>
+                    <field name="partner_longitude" invisible="1"/>
+                </xpath>
+            </field>
+        </record>
+
+        <record id="crm_lead_view_pivot_forecast" model="ir.ui.view">
+            <field name="name">crm.lead.view.pivot.geo_remove.inherit</field>
```

Comment:
A `geo_remove` slipped through. Let's use `forecast` and the same module suffix after `inherit` instead  :-)